python/ray/autoscaler/v2/sdk.py

```diff
@@ -1,13 +1,17 @@
-from typing import List
+from typing import List, Optional
 
 import ray
+from ray._raylet import GcsClient
+from ray.core.generated.experimental.autoscaler_pb2 import GetClusterStatusReply
 
 DEFAULT_RPC_TIMEOUT_S = 10
 
 
-def get_gcs_client():
-    """Get GCS client from the worker"""
-    return ray.worker.global_worker.node.get_gcs_client()
+def get_gcs_client(gcs_address: Optional[str] = None):
+    """Get the GCS client."""
+    if gcs_address is None:
+        gcs_address = ray.get_runtime_context().gcs_address
```

Comment:
Is ray.get_runtime_context().gcs_address guaranteed to be not None?

Reply:
Yeah, it would raise errors if so (which happens if ray.init() is not called, which is checked by the caller of this function as of now). 

Let me add comments maybe.